rti/mosaic-starter/src/main/java/org/eclipse/mosaic/starter/MosaicSimulation.java

```diff
@@ -169,12 +169,12 @@ public SimulationResult runSimulation(Path scenarioDirectory, CScenario scenario
 
             Validate.isTrue(Files.exists(scenarioDirectory), "Scenario directory at '" + scenarioDirectory + "' does not exist.");
 
-            scenarioId = Validate.notBlank(scenarioConfiguration.simulation.id,
+            federationId = Validate.notBlank(scenarioConfiguration.simulation.id,
                     "No simulation id given in scenario configuration file"
```

Comment:
Rename error message to "No scenario id given ..."

Reply:
Keep in mind, the `scenario_config.json` looks like:
```json
{
    "simulation": {
        "id": "Tiergarten",
        "duration": "60s",
        ...
```

Reply:
I know :( ok, then keep it as it is